src/platforms/android/enhance-errors/kotlin-compiler-plugin.mdx

```diff
@@ -17,7 +17,7 @@ The [Sentry Kotlin Compiler Plugin](https://github.com/getsentry/sentry-android-
 
 <Note>
 
-The Sentry Kotlin Compiler plugin is considered _experimental_. Give it a try and provide early feedback [on GitHub](https://github.com/getsentry/sentry-android-gradle-plugin).
+Because the Sentry Compiler plugin uses newer Kotlin Compiler features, the minimum supported Kotlin version is `1.8.20`.
```

Comment:
```suggestion
The minimum supported Kotlin version is `1.8.20`.
```

Don't think we need to include an explanation ("newer" will become outdated in the future anyway)